EdFi.Buzz.Installer/eng/Windows/AppSharedLibrary/Buzz-App-Install.psm1

```diff
@@ -308,6 +377,9 @@ $functions = @(
     "Install-NginxFiles"
     "Update-WebConfig"
     "Update-NginxConf"
+	"Redo-Site"
```

Comment:
Get-Verb will complain about Redo as a verb. Let's change this to Start-RedoSite.